Meta/import-wpt-test.py

```diff
@@ -81,23 +82,26 @@ def download_files(filepaths):
     downloaded_files = []
 
     for file in filepaths:
-        if (file.destination.exists()):
-            print(f"Skipping {file.destination} as it already exists")
+        source = urljoin(file.source, "/".join(file.source.split('/')[3:]))
+        destination = Path(file.destination).resolve()
+
+        if (destination.exists()):
```

Comment:
These parentheses are redundant.

Reply:
✅  Thanks — removed the unnecessary parens